diagnostic.js

```diff
@@ -30,25 +30,71 @@ mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+const create = (name, description, startYear, endYear) => {
+  Movement.create({
+    name: name,
+    description: description,
+    startYear: startYear,
+    endYear: endYear
+  }).then(function(movement){
+    console.log(movement.toJSON());
+  }).catch(console.error).then(done);
```

Comment:
`done`?

We had a done function on Tuesday to kill the database connection, but that was because we were writing an app that launched from the console, and ran one command per execution. We needed to kill the DB to prevent lots of open database connections from piling up.